docs/learning/presentation/index.md

```diff
@@ -1,37 +1,12 @@
 # iTwin.js Presentation Library
 
-## Purpose
+The Presentation library provides means to declaratively get data from iModels in a format that's ready to be displayed to end users. That includes creating hierarchical structures and content for components like list, table, property grid and others. See the [motivation page](./motivation/index.md) for more details.
 
-iTwin.js Presentation library helps retrieve presentation data from iModels and
-takes care of unified selection. What data to retrieve from an iModel is
-defined by consumers in a declarative way, using presentation rules - that's
-why the library is commonly called "presentation rules engine".
+[!bwc tile heading="Motivation" linkTo="index subPath=/presentation/motivation" contents="What is the purpose of the Presentation Library and why it should be used" icon="new.svg" step="13" width="28%"]
+[!bwc tile heading="Setting Up" linkTo="index subPath=/presentation/setup" contents="How to set up backend and frontend for using Presentation Library" icon="developer.svg" step="13" width="28%"]
 
```

Comment:
Users would benefit from numbering the tiles so they know what to look at first. I'd start at 0 for Motivation, because it is sort of an optional introductory chapter.

Reply:
Not sure. I agree that "Motivation" is the first page to read and then "Setting up". But afterwards it all depends on what you need.

Reply:
Sounds like a situation many non-fiction book authors face. In introduction they usually tell the reader that they laid out the chapters as best as they could, but expect people to skip to the parts they are interested in and return to earlier chapters if necessary.

If you are new to the topic, you may not know what to look at first, thus you trust the author's judgement to guide you through the material. It's has to do with human psychology: currently there is no guidance, so new readers must make a blind choice, which creates doubt and makes people frustrated.

Reply:
Sounds like a situation many non-fiction book authors face. In introduction they usually tell the reader that they laid out the chapters as best as they could, but expect people to skip to the parts they are interested in and return to earlier chapters if necessary.

If you are new to the topic, you may not know what to look at first, thus you trust the author's judgement to guide you through the material. It's has to do with human psychology: currently there is no guidance, so new readers must make a blind choice, which creates doubt and makes people frustrated.

Reply:
I don't agree there's no guidance - the pages that should be read first are at the top. In addition, They have headers and descriptions so I believe the reader knows what to expect.

I wouldn't be too opposed to adding the numbering if there were at least 3 numbered items. Now we only have "Motivation" and "Setting Up" which can be numbered "0" and "1". Numbering the rest makes no sense and it makes no sense to number just 2 items... Especially since they're already in left-to-right order and with headers and descriptions.

Reply:
Not all cultures read left-to-right and top-to-bottom...

This is why nobody uses grid layout to list documentation topics. A flat list is chosen instead, which inherently orders things.

The current grid-like pattern also isn't scalable. With each new topic you need to come up with an icon too.

Reply:
I personally don't see a problem with ordering due to the reasons explained above - there's no particular order that you should look at those pages.

I agree that maintaining a flat list instead of this grid is easier because of the icons, but we're not going to have many tiles. I can see at most a few more in there, so that's not a big deal. And displaying them as tiles provides a much better UX, in my opinion. E.g. [this](https://www.itwinjs.org/learning/tutorials/) VS [this](https://www.itwinjs.org/learning/guidelines/).

Reply:
I prefer flat list out of the two patterns because I can scan through the list much more quickly.

Regarding the UX, our tutorials landing page looks and functions well because tiles are divided into groups of four or less. If you want to follow this pattern, presentation learning landing page should also separate topics into categories, e.g. Introduction, Rules, Advanced. Notice that I made Advanced a category instead of a topic.